### DISCUSSION TITLE
Biblia versão android

### DISCUSSION THREAD

**Discussion by michafelipe9583** [rodolfogoulart/aletheia-core-model#30] [Ideas]
Acredito que assim como eu também tem inúmeras pessoas que utilizam Android e gostaria de ter o conhecimento que abrangem no app na palma da mão gostei muito do que li e vi no aplicativo
seria uma sugestão e me ponho a disposição de ideias pois faço faculdade de Publicidade e propaganda

**Comment by rodolfogoulart** [comment]
Olá @michafelipe9583 que bom ler seu feedback, fico muito feliz em ver as pessoas usando e como tem sido uma ferramenta muito boa para elas.

Quanto ao desenvolvimento para plataforma **Android**, eu tenho este tempo neste tempo iniciado o desenvolvimento voltado para **Tablets**, _com a ajuda de alguns usuários que contribuiram ao projeto fiz a compra de um tablet para o desenvovimento e teste_. Porém, ainda é muito inicial o estágio de desenvolvimento para esta plataforma e bem trabalhoso por conta da maneira que o app é contruido voltado para Desktop, mas me anima muito por conta da possibilidade de utilização do Tablet mais voltada para a ministração de estudos e pregação. Futuramente para esta plataforma Android e Desktop esta no cronograma o desenvolvimento de sincronização entre dispositivos, possibilitando assim que um estudo seja feito no Desktop e enviado para o Tablet.

**Um adendo:** O desenvolvimento para Android especificamente Celular ainda não é uma possibilidade, pelo menos a curto ou médio prazo, por conta da visão do projeto ser um app de uso para estudo. Indico neste momento o app MyBible na [PlayStore](https://play.google.com/store/apps/details?id=ua.mybible&pcampaignid=web_share), a base de dados do Aletheia é proveniente deste mesmo app.

Quanto ao uso do App, compartilhe aqui um pouco mais como tem sido a experiencia de uso, que funcionalidades tem lhe ajudado e que idéias de funcionalidade ou layout teve ao logo do uso.

**Comment by michafelipe9583** [comment]
Primeiro agradeço Rodolfo pelo o informativo acredito que cada dia mais
possa crescer ministerialmente como também o app que tem sido um meio de
estudo para inúmeras vidas
te desejo boa sorte nessa empreitada qualquer ajuda que precisar estarei a
disposição!!


Em seg., 25 de nov. de 2024 às 10:33, Rodolfo Goulart <
***@***.***> escreveu:

**Reply by rodolfogoulart** [reply]
@michafelipe9583 

Eu lancei estes dias uma versão Mobile (Tablet), se quiser fazer parte e testar o app

Nem todas funcionalidades estão otimizadas para tablet, mas estou iniciando o desenvolvimento para esta plataforma.

[link da playstore](https://play.google.com/store/apps/details?id=com.i53.aletheia.bible)
